test/TestLogger.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Tests
+{
+    public class TestLogger : ILogger
+    {
+        private readonly Func<string, LogLevel, bool> _filter;
```

Comment:
nit: as a convention in this code base, member variables should *not* be prefixed with underscores. Also, access to member variables should be prefixed with `this.`.